jsapp/js/account/plans/addOnList.module.scss

```diff
@@ -0,0 +1,65 @@
+@use '~kobo-common/src/styles/colors';
+@use 'scss/breakpoints';
+
+.header {
+  font-size: 1.125em;
```

Comment:
I'm not in favor of using `sizes` at this point except where it's needed. In theory, it *could* make it easier for us to switch to a better, actually responsive UI. In practice, it just means we're using pixels for sizing everywhere, which means a lot of other (S)CSS will need to change if we ever make `sizes` responsive. We have no good excuse for writing green-field code that's not responsive in 2023; we're just creating new technical debt.

In addition, using pixels for a font size presents some [accessibility issues](https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/). I'd be down to change this to `rem`, or add this `em` length to `sizes`, but I'd really need to be convinced to use pixels.

Reply:
Let's go with `rem`s then :)

Reply:
We might even add them to `sizes` as `$r12: rem($x12)`? Or just define the mixin ([this one](https://medium.com/@bhargav3shah/scss-convert-pixel-values-to-rem-using-functions-f1cef575edfd) or similar)

Reply:
Sounds good! I ended up naming the function `pxToRem()`, just to avoid confusion with `[rem()](https://developer.mozilla.org/en-US/docs/Web/CSS/rem)`.